recipes/simbody/yum_requirements.txt

```diff
@@ -0,0 +1,2 @@
+libXi
+libXmu
```

Comment:
Sorry, should have added I think we need a `-devel` postfix for both of these.

Also, please add `mesa-libGLU-devel` as `freeglut` requires it.